tests/conftest.py

```diff
@@ -2,10 +2,25 @@
 Module with global fixtures
 """
 import os
+import shutil
+import logging
 
 import pytest
 
 from sentinelhub import SHConfig
+from sentinelhub.testing_utils import get_input_folder, get_output_folder
+
+INPUT_FOLDER = get_input_folder(__file__)
+OUTPUT_FOLDER = get_output_folder(__file__)
+
+
+def pytest_configure(config):
+    shconfig = SHConfig()
+    for param in shconfig.get_params():
+        env_variable = param.upper()
+        if os.environ.get(env_variable):
+            setattr(shconfig, param, os.environ.get(env_variable))
+    shconfig.save()
```

Comment:
Fixtures cannot be called directly. However since `pytest_configure` saves the config, the config fixture does not need to read the environment variables, no?

Reply:
Yes, then let's read it from the saved config.